build/package/helm/gateway/files/policies/policies_test.rego

```diff
@@ -0,0 +1,20 @@
+package m8.authz
+
+alice_admin = {"User": {"Id": "1234", "Name": "alice", "Roles": [{"Name": "admin", "Scope": "system"}, {"Name": "admin", "Scope": "tenant", "Resource": "1234"}]}}
+
+bob_tenant_admin = {"User": {"Id": "12345", "Name": "bob", "Roles": [{"Name": "admin", "Scope": "tenant", "Resource": "1234"}]}}
+
+jane = {"User": {"Id": "123456", "Name": "jane"}, "Path": "/domain.CommandHandlerExtensions/"}
+
+scim_scope = {"Path": "/scim/something", "Authentication": {"Scopes": ["WRITE_SCIM"]}}
+
+test_system_admin {
+	is_system_admin with input as alice_admin
+	not is_system_admin with input as bob_tenant_admin
+}
+
+test_authorized {
+	authorized with input as alice_admin
+	authorized with input as jane
+	authorized with input as scim_scope
+}
```

Comment:
I think we should have a test case for `tenant_admin_rolebindings` that makes sure bob can only execute on 1234 and not any other resource.

Reply:
Done